react-redux-leaflet/React-Redux-Leaflet.md

```diff
@@ -373,6 +352,13 @@ const mapStateToProps = (state) => {
 export default connect(mapStateToProps)(Map)
 ```
 
+Then, get the adapted HERE TileLayers file:
+
+```bash
+cd Map
+curl 'https://gitlab.com/api/v4/projects/1345047/repository/files/leaflet-tilelayer-here.js/raw?ref=master' >> hereTileLayers.js
```

Comment:
It won't work with this file, the one provided in this repository is needed

Reply:
oh sorry, I forgot to change to our github repo. more elegant solution though. otherwise people have to clone the repo and copy manually. will amend..